bindings/python/.pylintrc

```diff
@@ -5,6 +5,13 @@ disable=missing-module-docstring,
 	invalid-name, # files that starts with number
 	duplicate-code,
 	consider-using-with,
-	consider-using-f-string
-
+	fixme, # TODOS 
+	too-many-instance-attributes,
+	too-many-arguments,
+	too-many-public-methods,
+	too-few-public-methods,
+	too-many-locals,
+	missing-function-docstring
```

Comment:
Hmm why? Is there an issue with generated code or something? Pretty annoying to disable this

Reply:
euh i think i added this after doing all the methods that werent removed in 2.0. to make merge easier.
Ill add a TODO maybe?

Reply:
But do we have that many missing docs in develop? I thought we'd be fine by now

Reply:
around 30 or so methods as_dict/from_dict that we replaced in 2.0

Reply:
added todo on it

Reply:
removed again